2-mandatory/2-financial-times.js

```diff
@@ -14,24 +22,59 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
-}
 
+    let smallestTitle = allArticleTitles[0];
+
+    for (i = 1; i < allArticleTitles.length; i++) {
```

Comment:
Issue: The iterator variable `i` in a for loop should be declared with `let`. For example
```js
for (let i = 1; i < allArtivlesTitles.length; i++)
```
Suggestion: Use a for...of loop instead of a for loop to improve readability